depobs/website/static/index.html

```diff
@@ -4,6 +4,7 @@
       <meta charset="utf-8">
       <meta http_equiv="Content_Security_Policy" content="default_src 'self'; connect_src https://depobs.dev.mozaws.net/;">
       <title id="title">Mozilla Dependency Observatory</title>
+      <base href="static/">
```

Comment:
@psiinon this is kind of a hack to fix 404s. We could switch to URLs relative to root too.